src/main/java/repositories/CorrectInfoComparisonException.java

```diff
@@ -0,0 +1,8 @@
+package repositories;
+
+public class CorrectInfoComparisonException extends Exception {
```

Comment:
You can make it unchecked exception to avoid handling in menu 😈

Reply:
done